webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -161,11 +165,13 @@ const Input = React.forwardRef((
             return (
                 <textarea
                     ref={ref as React.RefObject<HTMLTextAreaElement>}
-                    id={`input_${name || ''}`}
+                    id={inputId}
                     className={classNames('Input form-control', inputSize, inputClassName, {Input__focus: showLegend})}
                     value={value}
                     placeholder={placeholderValue}
                     aria-label={ariaLabel}
+                    aria-describedby={error ? errorId : undefined}
```

Comment:
Looking around for how to do this, we should be using `aria-errormessage` for this instead of `aria-describedby`. [This example](https://w3c.github.io/aria/#example-37) uses `aria-live` instead of an `alert` role, but I tested the `alert` role locally, and it seems to work

Reply:
We discussed this internally (https://hub.mattermost.com/private-core/pl/oemaj6qn33rb8nipqxgkyg4qpo), and based on the feedback from LA and the somewhat-limited support for `aria-errormessage` from what we can find, we're going to stick with this pattern for form errors